src/main/java/org/alfresco/repo/search/impl/querymodel/impl/db/DBQueryBuilderPredicatePartCommand.java

```diff
@@ -198,5 +201,15 @@ else if(function == LuceneFunction.UPPER)
             return alias +"." +fieldName;
         }
     }
+
+	public void setPropertyQName(QName propertyQName) 
+	{
+		this.propertyQName = propertyQName;
+	}
+	
+	public QName getPropertyQName() 
+	{
+		return this.propertyQName;
+	}

```

Comment:
Use spaces instead of tabs.

Reply:
Sorted out in the last [commit](https://github.com/Alfresco/alfresco-repository/pull/1204/commits/c0752a79701821c01f98dc332bc8e673391a04aa).